### DISCUSSION TITLE
State management - typescript

### DISCUSSION THREAD

**Discussion by itester101** [badeball/cypress-cucumber-preprocessor#982] [Q&A]
Hello, 

In the [documenation](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/state-management.md) in Typescript section examples, there is no implementation of the `eat(count: number): void` method. Where do I put it? 

Thanks in advance

**Comment by fredjammes** [comment]
I'd say you have to create a step, or maybe a BeforeAll in which you define eat(count: number), such as : 

```
BeforeAll(function (this: CustomWorld) {
  this.eat = (count: number) => { 
    // do something with count
  };
});
```
